src/onig-scanner.cc

```diff
@@ -58,8 +58,8 @@ OnigScanner::OnigScanner(Local<Array> sources) {
   regExps.resize(length);
 
   for (int i = 0; i < length; i++) {
-    String::Utf8Value utf8Value(sources->Get(i));
-    regExps[i] = shared_ptr<OnigRegExp>(new OnigRegExp(string(*utf8Value)));
+    Local<String> source = Local<String>::Cast(sources->Get(i));
+    regExps[i] = shared_ptr<OnigRegExp>(new OnigRegExp(OnigString(source)));
```

Comment:
Yeah, this is my first PR on this repo, and I noticed we had a bunch of `using` statements in headers. I removed a couple of them in files that I touched, but I don't think I want to update the styling of the rest of the code at the moment.